src/lsm/table.zig

```diff
@@ -332,9 +332,9 @@ pub fn TableType(
                 errdefer allocator.free(data_block);
 
                 return Builder{
-                    .index_block = index_block[0..block_size],
-                    .filter_block = filter_block[0..block_size],
-                    .data_block = data_block[0..block_size],
+                    .index_block = index_block,
+                    .filter_block = filter_block,
+                    .data_block = data_block,
```

Comment:
I believe the original intention of the comptime-known slice here is to upgrade type safety, to an array over a slice?

Can you help me understand the reason for dropping the slice now (Zig 0.11)?

Reply:
Thanks for the excellent explanation. 🕺